tests/e2e/network/service_annotations.go

```diff
@@ -195,6 +196,80 @@ var _ = Describe("Service with annotation", func() {
 		Expect(err).NotTo(HaveOccurred())
 		Expect(ret).To(Equal(true), "external ip %s is not in the target subnet %s", ip, newSubnetCIDR)
 	})
+	It("can be bound to the mode of load balancer", func() {
```

Comment:
nit: change to `"should bound to the specified load balancer from azure-load-balancer-mode"`